about.html

```diff
@@ -101,42 +101,44 @@ <h3>
         <p>
             Take a look at the last two conferences held in Nigeria and Kenya.
         </p>
-        <div class="Third-section2-img">
-            <img src="./Images/cdc-events.jpg" alt="">
-            <span class="conference-year">2017</span>
-            <span class="conference-venue">CPHIA2021 Global Summit in Nigeria</span>
-        </div>
-        <div class="Third-section2-img">
-            <img src="./Images/cdc-events2.jpg" alt="">
-            <span class="conference-year">2019</span>
-            <span class="conference-venue">CPHIA2021 Global Summit in Kenya</span>
+        <div class="pastEvents">
+            <div class="Third-section2-img1">
+                <img src="./Images/cdc-events.jpg" alt="past-event-img">
+                <span class="conference-year">2017</span>
+                <span class="conference-venue">CPHIA2021 Global Summit in Nigeria</span>
+            </div>
+            <div class="Third-section2-img2">
+                <img src="./Images/cdc-events2.jpg" alt="past-event-img">
+                <span class="conference-year">2019</span>
+                <span class="conference-venue">CPHIA2021 Global Summit in Kenya</span>
+            </div>
         </div>
```

Comment:
- [ ] As requested by the previous reviewer, according to the requirements of this project the two elements should be aligned horizontally in the desktop version. This can be achieved by setting the `.pastEvents { display: flex }`

Reply:
Hi @Cathella. As requested I have made the two elements align horizontal on the desktop version